docs/security/fips_support.md

```diff
@@ -6,15 +6,15 @@ FIPS 140-2 is a U.S. Federal Government security standard used to approve crypto
 
 ## Use of FIPS Compatible Go compiler.
 
-The Go compiler in use can be found [here](https://hub.docker.com/u/goboring). Each component of the system is built with the version of this compiler that matches the same standard Go compiler version that would be used otherwise.
+The Go compiler in use can be found [here](https://go.googlesource.com/go/+/dev.boringcrypto). Each component of the system is built with the version of this compiler that matches the same standard Go compiler version that would be used otherwise.
 
 This version of Go replaces the standard Go crypto libraries with the FIPS validated BoringCrypto module. See the [readme](https://github.com/golang/go/blob/dev.boringcrypto/README.boringcrypto.md) for more details.
 
 Moreover, this module is currently being [revalidated](../assets/fips_engagement.pdf) as the Rancher Kubernetes Cryptographic Library for the additional platforms and systems supported by RKE2.
 
 ### FIPS Support in Cluster Components
 
-Most of the components of the RKE2 system are statically compiled with the GoBoring Go compiler implementation that takes advantage of the BoringSSL library. RKE2, from a component perspective, is broken up in a number of sections. The list below contains the sections and associated components.
+Most of the components of the RKE2 system are statically compiled with the GoBoring Go compiler implementation. RKE2, from a component perspective, is broken up in a number of sections. The list below contains the sections and associated components.
```

Comment:
This deletion clears up an inconsistency with "This version of Go replaces the standard Go crypto libraries with the FIPS validated BoringCrypto module." sentence two paragraphs earlier.

Reply:
The reasons that this is significant is that BoringSSL also uses the BoringCrypto module but BoringSSL is not distributed as part of RKE2 and RKE2 is not build with BoringSSL.